### DISCUSSION TITLE
having an issue when truing to add a collaborators to the private repo.

### DISCUSSION THREAD

**Discussion by goqket** [community/community#81206] [New to GitHub]
hello all,

I am new to github. I recently created a private repo, but whenever I try to add a collaborator to it, I get "User could not be added".
Seems like issue happen whenever I try to invite an existing user, whenever I send an invite to an email that does not hav e a GitHub account, invite gets sent, but they never get am invitation email.
One more thing, I also cannot add my account to an Org, 
<img width="2273" alt="image" src="https://github.com/community/community/assets/153666694/c6b91d6b-2920-46f9-919b-e24c5373e266">
<img width="1253" alt="image" src="https://github.com/community/community/assets/153666694/dd90c0a1-6e98-4f87-a77f-bca994ad5514">
<img width="2288" alt="image" src="https://github.com/community/community/assets/153666694/98a1d620-4c17-42da-bbf9-0b76570192ba">

**Comment by queenofcorgis** [comment]
Thanks for your patience, @goqket ❤️ 

May I suggest checking out [this discussion](https://github.com/orgs/community/discussions/35582) first? I'm not sure if that could be the issue, but a good first place to look.

**Comment by abhishekraaut** [comment]
please give solution After I created a Repository on Github and tried to invite collaborators, I am shown the message 'User could not be added' .

**Comment by ProyectoTuTutorPUJ** [comment]
It looks like your account is an organization one, it means GitHub recognizes you as an a group of people instead of individual user.
I solved by changing the name of my user profile, and then changed it again to the original one.
